src/frontend/ocamlmerlin/query_json.ml

```diff
@@ -1,442 +1,424 @@
 (* {{{ COPYING *(
 
-  This file is part of Merlin, an helper for ocaml editors
+     This file is part of Merlin, an helper for ocaml editors
 
-  Copyright (C) 2013 - 2015  Frédéric Bour  <frederic.bour(_)lakaban.net>
-                             Thomas Refis  <refis.thomas(_)gmail.com>
-                             Simon Castellan  <simon.castellan(_)iuwt.fr>
+     Copyright (C) 2013 - 2015  Frédéric Bour  <frederic.bour(_)lakaban.net>
+                                Thomas Refis  <refis.thomas(_)gmail.com>
+                                Simon Castellan  <simon.castellan(_)iuwt.fr>
 
-  Permission is hereby granted, free of charge, to any person obtaining a
-  copy of this software and associated documentation files (the "Software"),
-  to deal in the Software without restriction, including without limitation the
-  rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
-  sell copies of the Software, and to permit persons to whom the Software is
-  furnished to do so, subject to the following conditions:
+     Permission is hereby granted, free of charge, to any person obtaining a
+     copy of this software and associated documentation files (the "Software"),
+     to deal in the Software without restriction, including without limitation the
+     rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+     sell copies of the Software, and to permit persons to whom the Software is
+     furnished to do so, subject to the following conditions:
 
-  The above copyright notice and this permission notice shall be included in
-  all copies or substantial portions of the Software.
+     The above copyright notice and this permission notice shall be included in
+     all copies or substantial portions of the Software.
 
-  The Software is provided "as is", without warranty of any kind, express or
-  implied, including but not limited to the warranties of merchantability,
-  fitness for a particular purpose and noninfringement. In no event shall
-  the authors or copyright holders be liable for any claim, damages or other
-  liability, whether in an action of contract, tort or otherwise, arising
-  from, out of or in connection with the software or the use or other dealings
-  in the Software.
+     The Software is provided "as is", without warranty of any kind, express or
+     implied, including but not limited to the warranties of merchantability,
+     fitness for a particular purpose and noninfringement. In no event shall
+     the authors or copyright holders be liable for any claim, damages or other
+     liability, whether in an action of contract, tort or otherwise, arising
+     from, out of or in connection with the software or the use or other dealings
+     in the Software.
 
-)* }}} *)
+   )* }}} *)
```

Comment:
We do not ocamlformat the codebase yet, and formating changes should generally not be mixed in unrelated PRs.
I only ocamlformated your `syntax_command` new file because it would have been very cumbersome to do it manually, but you should not format other files with it.

Reply:
Oh my bad. Got it.

Reply:
This is not resolved yet, the formatting is still wrong :-)

Reply:
Oh, just pushed it in the new commit. 👍